src/webrtc/VegaRtcManager.js

```diff
@@ -1452,6 +1474,27 @@ export default class VegaRtcManager {
         }
     }
 
+    _onConsumerScore({ consumerId, kind, score }) {
+        this._logger.debug("_onConsumerScore()", { consumerId, kind, score });
+        const consumer = this._consumers.get(consumerId);
+        const { sourceClientId } = consumer.appData;
+
+        if (consumer && sourceClientId) {
```

Comment:
Would possibly rewrite this to: 

```suggestion
        const { sourceClientId } = this._consumers.get(consumerId) || { appData: {} };

        if (sourceClientId) {
```